api/src/main/java/com/velocitypowered/api/event/player/ServerConnectedEvent.java

```diff
@@ -12,10 +14,19 @@
 
   private final Player player;
   private final RegisteredServer server;
+  private final @Nullable RegisteredServer previousServer;
 
-  public ServerConnectedEvent(Player player, RegisteredServer server) {
```

Comment:
API breakage. Readd this constructor and simply forward to the new one.

Reply:
Should I mark it as deprecated?

Reply:
That's fine.